bundles/org.eclipse.jface.text/src/org/eclipse/jface/text/contentassist/AsyncCompletionProposalPopup.java

```diff
@@ -113,10 +113,7 @@ public String getDisplayString() {
 
 		@Override
 		public String getAdditionalProposalInfo() {
-			 return NLS.bind(JFaceTextMessages.getString("AsyncCompletionProposalPopup.computingDetails"), new Object[] { //$NON-NLS-1$;
-				Integer.valueOf(fSize),
-				Integer.valueOf(fSize - fRemaining),
-				Integer.valueOf(fRemaining) });
+			 return NLS.bind(JFaceTextMessages.getString("AsyncCompletionProposalPopup.computingDetails"), Integer.valueOf(fSize), Integer.valueOf(fSize - fRemaining), Integer.valueOf(fRemaining));
```

Comment:
@jjohnstn As you're so fast, here is a NON-NLS comment being removed by the save actions.

Reply:
Continues in https://github.com/eclipse-platform/eclipse.platform.ui/pull/3293/files#r2384176756 .